### ISSUE TITLE
Get rid of intermittent battleship grey background pages

### ISSUE THREAD

**Issue by natechadwick** [percussion/percussioncms#992]
**Describe the bug**
The page background #6C717C should have been removed a long time ago.  Get rid of any css declarations calling for background: #6C717C at the body level.
